src/open_inwoner/accounts/templates/registration/login.html

```diff
@@ -49,7 +49,7 @@ <h1 class="h1">{% trans 'Welkom' %}</h1>
 
                 {% if login_allow_registration %}
                     {% render_card tinted=True %}
-                        {% render_form id="login-form" method="POST" form=form %}
+                        {% render_form id="login-form" method="POST" form=form show_required=True %}
```

Comment:
I think we need something `align-items: center;` here?


![error-login](https://user-images.githubusercontent.com/63058071/229720341-0eb82e97-78b3-429c-aa7b-8d8ab4292486.png)